portfolio/src/main/webapp/results.js

```diff
@@ -0,0 +1,24 @@
+// Extract parameters from the URL (may use a Servlet to fetch details later).
+const urlParams = new URLSearchParams(window.location.search);
+const paramNames = {
```

Comment:
Thought: Should this be a in a constants JS file?

Reply:
Added, thanks for the suggestion!